pyproject.toml

```diff
@@ -43,6 +43,7 @@ dagster-postgres = "^0.22.11"
 alembic = "^1.13.1"
 python-dateutil = "^2.9.0.post0"
 chardet = "^5.2.0"
+liia-tools-pipeline-config = {git = "https://github.com/SocialFinanceDigitalLabs/liia-tools-pipeline-config.git"}
```

Comment:
Will this have knock-on effects for the Dockerfiles @cyramic ?

Reply:
It shouldn't as it uses this file to determine what to load.

Reply:
Hmm but similar to how the github workflow needed permission to run the tests which involves installing this package, won't the Dockerfile also need permission to install this package?

Reply:
Hmmm.  Fair point. The config repo is private, but is being run by the organisation account when the docker image is built.  There should be an easy way to test this though. I can do a test build and see what happens?